core/src/main/java/hudson/util/RingBufferLogHandler.java

```diff
@@ -23,9 +23,6 @@
  */
 package hudson.util;
 
-import org.kohsuke.accmod.Restricted;
```

Comment:
can we add a checkstyle rule to prevent re-occurrence please?

Reply:
Declined, the scope of this PR is fixing existing violations, not adding automated enforcement. Automated enforcement can be done in a separate PR.